src/components/Footer.astro

```diff
@@ -2,24 +2,89 @@
 const currentYear = new Date().getFullYear();
 ---
 
-<footer>
-  &copy; {currentYear} Sasquatch Devs
-  <small class="byline">Built by Astro</small>
+<footer class="bg-gray-100 flex justify-center items-center">
+  <div class="flex flex-row flex-wrap justify-center items-center m-2 p-3 text-white bg-[url('/assets/3Dwave.jpg')] bg-center bg-cover bg-no-repeat rounded-3xl 2xl:w-7/12 lg:w-full lg:p-10 lg:my-10 md:mx-5">
```

Comment:
This class `2xl:w-7/12` makes the footer smaller than the other sections, `max-w-screen-2xl` should do it.